thrust/thrust/detail/type_traits/is_thrust_pointer.h

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2008-2020 NVIDIA Corporation
```

Comment:
I love our license

Reply:
Regarding the licenses I added at the start of the new files, is there any guidance on how to select the appropriate license? I think I just used the licenses in the existing files located in the same directories.

Reply:
The rule given to me by @jrhemstad was to look at the neighboring files and copy what's there, until we figure out a better story.